lib/src/dart_cli.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:universal_io/io.dart';
+
+/// Dart CLI
+class Dart {
+  /// Install dart dependencies (`dart pub get`).
+  static Future<void> packagesGet([String? cwd]) {
+    return _Cmd.run('dart', ['pub', 'get'], workingDirectory: cwd);
+  }
+
+  /// Determine whether dart is installed
+  static Future<bool> installed() async {
+    try {
+      await _Cmd.run('dart', []);
+      return true;
+    } catch (_) {
+      return false;
+    }
+  }
+}
+
+class _Cmd {
```

Comment:
since this code is the exact same across the flutter_cli and dart_cli can we extract and reuse it?